### PR TITLE
Fix the typo of FusedRMSNorm doc

### PR DESCRIPTION
**Pull request by cqulilujia**
Fix the typo of FusedRMSNorm doc in normalization/fused_layer_norm.py

### CODE REVIEW

File: apex/normalization/fused_layer_norm.py [hunk 1]
```diff
@@ -745,8 +745,7 @@ class FusedLayerNorm(torch.nn.Module):
             normalize over the last dimension which is expected to be of that specific size.
         eps: a value added to the denominator for numerical stability. Default: 1e-5
         elementwise_affine: a boolean value that when set to ``True``, this module
-            has learnable per-element affine parameters initialized to ones (for weights)
-            and zeros (for biases). Default: ``True``.
+            has learnable per-element affine parameters initialized to ones for weights. Default: ``True``.
```
Comment (crcrpar) [inline apex/normalization/fused_layer_norm.py:748]:
```suggestion
            has learnable per-element affine parameters initialized to ones for weights. Default: :obj:`True`.
```

**Review by crcrpar — APPROVED** [review]
Thank you for the fix

**Review by AumGupta — APPROVED** [review]

File: apex/normalization/fused_layer_norm.py (hunk 1 above)
Comment (cqulilujia) [reply apex/normalization/fused_layer_norm.py]: Submitted a newer version:
1. use :obj: as recommended
2. fix the previous pr's issue(mistakenly modified FusedLayerNorm rather than FusedRMSNorm )

**Review by crcrpar — COMMENTED** [review]
thank you!
